scripts/verify-spelling.sh

```diff
@@ -60,7 +63,9 @@ cd "${ROOT}"
 RES=0
 echo "Checking spelling..."
 ERROR_LOG="${TMP_DIR}/errors.log"
-git ls-files | grep content/${LANGUAGE} | xargs misspell > "${ERROR_LOG}"
+skipping_file="${KUBE_ROOT}/scripts/.spelling_failures"
```

Comment:
It is better to write the reason why the script needs to skip those files as https://github.com/kubernetes/website/pull/28954#pullrequestreview-707216934
Otherwise someone will fix those files by removing lines from .spelling_failures file in the future.

Reply:
Thanks for the suggestion. I've added some comments to this file.

Reply:
Thanks for updating. The additional `NOTE` seems good for me :)